android/src/ch/opengis/qfield/QFieldProjectActivity.java

```diff
@@ -140,11 +140,16 @@ private void drawView() {
                 for (File file : list) {
                     if (file.getName().startsWith(".")) {
                         continue;
-                    }else if (file.getName().toLowerCase().endsWith(".qgs") || file.getName().toLowerCase().endsWith(".qgz") ||
-                              file.getName().toLowerCase().endsWith(".gpkg") || file.getName().toLowerCase().endsWith(".shp") || 
-                              file.getName().toLowerCase().endsWith(".kml") || file.getName().toLowerCase().endsWith(".kmz") || 
-                              file.getName().toLowerCase().endsWith(".tif") || file.getName().toLowerCase().endsWith(".jpg") || 
-                              file.getName().toLowerCase().endsWith(".png") || file.getName().toLowerCase().endsWith(".pdf") ||
+                    }else if (file.getName().toLowerCase().endsWith(".qgs") ||
+                              file.getName().toLowerCase().endsWith(".qgz") ||
+                              file.getName().toLowerCase().endsWith(".gpkg") || 
+                              file.getName().toLowerCase().endsWith(".shp") || 
+                              file.getName().toLowerCase().endsWith(".kml") || 
+                              file.getName().toLowerCase().endsWith(".kmz") || 
+                              file.getName().toLowerCase().endsWith(".tif") || 
+                              file.getName().toLowerCase().endsWith(".jpg") || 
+                              file.getName().toLowerCase().endsWith(".png") || 
+                              file.getName().toLowerCase().endsWith(".pdf") ||
                               file.getName().toLowerCase().endsWith(".gpx")){
```

Comment:
Just noticed this is in java, lol. Still, can you make a similar list for java, so adding a new file extensions can simply be modifiying the list in java and c++.

Reply:
We'd only ever use the list here on the java side so I think ATM the list would be a slight overkill. I also just hate coding java :smile: